lib/protobuf/field/bytes_field.rb

```diff
@@ -48,7 +48,18 @@ def wire_type
 
       def coerce!(value)
         case value
-        when String, Symbol
+        when String
+          if value.encoding == Encoding::ASCII_8BIT
```

Comment:
Would it make sense to make this configurable at the `from_json` method? You can specify if your uses Base64 encoded binary blobs? Maybe true by default but allow the caller to turn it off? That way we don't have to guess.